src/NodeApi.Generator/TypeDefinitionsGenerator.cs

```diff
@@ -666,7 +666,8 @@ private string GetTSParameters(ParameterInfo[] parameters)
         foreach (ParameterInfo p in parameters)
         {
             string parameterType = GetTSType(p);
-            s.AppendLine($"{TSIdentifier(p.Name)}: {parameterType},");
+            string optionalToken = p.IsOptional ? "?" : "";
+            s.AppendLine($"{TSIdentifier(p.Name)}{optionalToken}: {parameterType},");
```

Comment:
This change is needed also at line 659 above. That formats a single parameter on the same line as the method, while multiple parameters here are formatted each on separate lines.

Reply:
Fixed! A lot of parameters are coming through like `orgId?: string | null`. Is that how we want to handle optional and nullable?

Reply:
Probably not. I think optional parameters should just use `?` without the `| null`. Then if/when JS passes an `undefined` value (or omits the argument) it gets implicitly converted to `null` in .NET.

See also [the issue tracking null/undefined confusion](https://github.com/microsoft/node-api-dotnet/issues/58). Early in the project we were unsure (and debated) how to handle some things. Now that it's further along I think we can do a more complete analysis and decide on some consistent patterns.

Reply:
The `| null` issue can be addressed separately, so I'll resolve this issue and merge the PR.